packages/devops/scripts/lambda/utilities.py

```diff
@@ -32,7 +36,6 @@ def tags_contains(tags, key, value):
     tag_matching_key = tags_matching_key[0]
     return tag_matching_key['Value'] == value
 
-
```

Comment:
Minor: do you have python lint? The two blank lines between top level functions comes from there https://www.python.org/dev/peps/pep-0008/#blank-lines
Pretty handy to have anyway, caught a bunch of issues for me while working on this last

Reply:
Oh I don't! Will add it

Reply:
Actually it looks like I do, but it's not picking up the two lines thing. Do you know what config I might need to tweak?
![image](https://user-images.githubusercontent.com/1274422/129664254-de0be978-f829-4b6e-a776-66fc3b1584de.png)

Reply:
Hmm not sure, this is what I see using PyCharm CE

<img width="796" alt="Screen Shot 2021-08-17 at 2 36 43 pm" src="https://user-images.githubusercontent.com/667652/129664536-75283125-9ca0-4adc-aedf-bb36909ce587.png">

Reply:
Huh...yeah I've just added the double line break manually.